common/crypto/attestation-api/common/CMakeLists.txt

```diff
@@ -8,15 +8,15 @@ SET(JWT_PATCHED_FILEPATH ${CMAKE_CURRENT_SOURCE_DIR}/${JWT_PATCHED_FILENAME} PAR
 
 ADD_CUSTOM_COMMAND(
 	OUTPUT ${JWT_PATCHED_FILENAME}
-	COMMAND [ ! -f ${JWT_PATCHED_FILENAME} ] && cd jwt-cpp && git apply ../jwt-cpp.patch && touch ../${JWT_PATCHED_FILENAME} || true
+    COMMAND [ ! -f ${JWT_PATCHED_FILENAME} ] && cd jwt-cpp && patch -f -p1 < ../jwt-cpp.patch && touch ../${JWT_PATCHED_FILENAME} || true
 	WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
 	)
 
 ADD_CUSTOM_TARGET(patch_jwt DEPENDS ${JWT_PATCHED_FILENAME})
 
 # TODO: devise better strategy for clean up; usually the build folder is removed so the clean is not called
 ADD_CUSTOM_TARGET(clean_patch_jwt
```

Comment:
actually what you did with the OUTPUT above should add the file to be cleaned automatically.  have you tried "make -C build clean"?

Reply:
This is tricky because we usually get rid of the build folder. I left a comment that a better strategy is needed for patching/unpatching